### DISCUSSION TITLE
Ability to hide the path

### DISCUSSION THREAD

**Discussion by slaFFik** [spatie/ray#653] [Ideas]
I would really appreciate it if it was possible to hide the path where the Ray app thinks the `ray()` call occurred.
Sometimes this info is irrelevant and always has the same value, regardless of the file/line I use `ray()` in: `...\wp-content\plugins\spatie-ray\vendor\autoload.php:99`.

I think this can be turned off either by setting a value in `ray.php` or inside the Ray app configuration, it doesn't matter.

**Comment by tonyrumans** [comment]
I just posted something similar. Have you figured anything else out about what is going on? https://github.com/spatie/ray/discussions/717
